disruption_py/machine/cmod/efit.py

```diff
@@ -133,12 +116,9 @@ def get_efit_parameters(params: PhysicsMethodParams):
             print("unable to get V_surf")
             efit_data["v_surf"] = np.full(len(efit_time), np.nan)
 
-        # For shots before 2000, adjust units of aminor, compute beta_n and v_loop
+        # For shots before 2000, compute beta_n and v_loop
         if params.shot_id <= 1000000000:
 
-            # Adjust aminor units
-            efit_data["aminor"] = efit_data["aminor"] / 100  # [cm] to [m]
```

Comment:
this was probably a bug, but let's double check because `aminor` was already in meters!
if the code downstream from this line was expecting it in meters, it was effectively getting it in... hundreds of meters. :confused:

Reply:
So before my changes:

- For pre-2000 shots, the function gets a_minor from '\efit_aeqdsk:aout' in [cm) and then divides the signal by 100 ([cm] -> [m])
- For post-2000 shots, the function gets a_minor from '\efit_aeqdsk:aminor' which is already in [m]

It's somewhat unnecessary but it worked.